HealthcareSystem/PatientWebApp/wwwroot/js/patient_examinations.js

```diff
@@ -84,8 +96,61 @@ $(document).ready(function () {
     });
 
     $('#btn_close').click(function () {
+
```

Comment:
Shouldn't you generate an end of scheduling here as well? Because if I clicked the Close button, it means that I finished my scheduling attempt and failed to schedule an appointment.

Reply:
Added in ab6f6987a1f4083ac09985d4d463121d1f5f4f37.